packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -116,6 +121,34 @@ export class OrganizationAPI extends AbstractAPI {
     });
   }
 
+  public async updateOrganizationLogo(organizationId: string, params: UpdateLogoParams) {
```

Comment:
:upside_down_face: Do we need to mention "organization" in the method name? We should avoid stuttering if we can.

What's the usage?

```ts
organizations.updateLogo()
```

Reply:
I agree with you, although all the methods in this file include the word `Organization`. It would be consistent if this one omits it.

Reply:
You're right, I need :eyeglasses: :smile: